in_text/text_data_release.yml

```diff
@@ -206,7 +206,7 @@ update: none planned
 themekeywords: ["process guided deep learning", "generalization", "explainable AI (XAI)", "stream temperature", "domain shift", "machine learning", "hybrid modeling", "water","temperature","modeling","climate change"]
 
 usage-rules: >-
-  These data are open access usable via creative commons as long as original data providers are acknowledged
+  None
```

Comment:
I think everything we release as the government is under a “public domain” license – while also open, not quite the same as creative commons. When you suggest that original data providers are acknowledged is this a requirement that they be cited for derivative works? Here are some ideas of use constraints that I think might be more appropriate for this release: (1) “None”, (2) “Users are advised to read the associated publication thoroughly to understand appropriate use and data limitations.”, (3) “If using input data for this model, users are advised to check the data source licensing to understand appropriate use.” I think “none” is the response I’ve seen the most in the releases I’ve looked over.